src/bidiMapper/modules/context/BrowsingContextImpl.ts

```diff
@@ -1530,7 +1530,12 @@ export class BrowsingContextImpl {
                 }
               }
 
-              startNodes = startNodes.length > 0 ? startNodes : [document.body];
+              startNodes =
+                startNodes.length > 0
+                  ? startNodes
+                  : Array.from(document.documentElement.children).filter(
```

Comment:
IDK if anything in `head` have a11y selectors.

Reply:
is this filtering in the spec?

Reply:
Yes:
> 5. Let child nodes be an empty [list](https://infra.spec.whatwg.org/#list) and, for each node child in the [children (https://dom.spec.whatwg.org/#concept-tree-child) of context node:
>     1. If child implements [Element](https://dom.spec.whatwg.org/#element), [append](https://infra.spec.whatwg.org/#list-append) child to child nodes.